contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -1837,6 +1853,12 @@ pltsql_sequence_datatype_map(ParseState *pstate,
 		ereport(WARNING,
 				(errmsg("NUMERIC or DECIMAL type is cast to BIGINT")));
 	}
+
+	if (aclresult != ACLCHECK_OK)
+    	aclcheck_error_type(aclresult, *newtypid);
```

Comment:
Why are we not throwing the error just after permission check on line 1766?
Fix indentation.

Reply:
I observed and figured that it is logically incorrect; I've made the necessary changes in the newer commits.